wolfcrypt/src/asn.c

```diff
@@ -15424,8 +15424,16 @@ static int ConfirmSignature(SignatureCtx* sigCtx,
                             ERROR_OUT(MEMORY_E, exit_cs);
                         }
                     #endif
-                        mp_init(r);
-                        mp_init(s);
+                        if((ret = mp_init(r)) != 0) {
```

Comment:
It might be easier, safer to just use:

```
if ((ret = mp_init_multi(r, s, NULL, NULL, NULL, NULL)) != MP_OKAY){
    goto exit_cs;
}
```

I think it is excessive to put the WOLFSSL_MSG and WOLFSSL_ERROR_VERBOSE for this. It can only fail for memory allocation reasons.